_artifacts/document-attributes.adoc

```diff
@@ -154,7 +155,7 @@ endif::OP[]
 :GETTING_STARTED_CASES: Getting started with case management
 :GETTING_STARTED_PROCESSES: Getting started with business processes
 :GETTING_STARTED_PLANNER: Getting started with {PLANNER}
-:CREATING_SPRING_BOOT_APPLICATIONS: Creating {PRODUCT} business applications with Spring Boot
+:CREATING_SPRING_BOOT_APPLICATIONS: Using {PLANNER} with Spring Boot to create a school timetable
```

Comment:
I think this one is wrong. CREATING_SPRING_BOOT_APPLICATIONS should refer to a different assembly, namely assembly-springboot-business-apps.adoc . The assembly that you edit, and the one about Planner, is assembly-business-optimizer-springboot.adoc .